bedrock/privacy/templates/privacy/archive/firefox-2008-06.html

```diff
@@ -37,8 +37,8 @@ <h1 itemprop="name">Mozilla Firefox Privacy Policy</h1>
             <dl>
-              <dd><span style="text-decoration: underline;">Firefox 1.0 – 2.x</span>. For these earlier versions of Firefox, “Talkback” is Firefox’s crash reporting feature. Talkback also collects Personal Information (including your name, email address) and Potentially Personal Information (including your IP address, your computer’s name, and the processes you were running at the time of the crash). You can selectively disable the sending of this information. Additionally, you have the option to include the URL of the site you were visiting when Firefox crashed, a comment, and your email address in the report. Mozilla only makes Non-Personal Information and Potentially Personal Information in the public reports available online at <a href="http://www.talkback-public.mozilla.org/">www.talkback-public.mozilla.org/</a>.</dd>
-              <dd><span style="text-decoration: underline;">Firefox 3.0 to 3.x</span>. For the current versions of Firefox, the Firefox Crash Reporter is Firefox’s crash reporting feature. With this feature, you have the option to include the URL of the site you were visiting when Firefox crashed, a comment, and your email address in the report. Firefox Crash Reporter also sends Potentially Personal Information to Mozilla in the form of a unique numeric value to distinguish individual Firefox installs. Mozilla only makes Non-Personal Information and Potentially Personal Information in the public reports available online at <a href="http://crash-stats.mozilla.com/">http://crash-stats.mozilla.com/</a>.</dd>
+              <dd><span class="underline">Firefox 1.0 – 2.x</span>. For these earlier versions of Firefox, “Talkback” is Firefox’s crash reporting feature. Talkback also collects Personal Information (including your name, email address) and Potentially Personal Information (including your IP address, your computer’s name, and the processes you were running at the time of the crash). You can selectively disable the sending of this information. Additionally, you have the option to include the URL of the site you were visiting when Firefox crashed, a comment, and your email address in the report. Mozilla only makes Non-Personal Information and Potentially Personal Information in the public reports available online at <a href="http://www.talkback-public.mozilla.org/">www.talkback-public.mozilla.org/</a>.</dd>
```

Comment:
Also, the `<u>` element is back in HTML5 (after being removed in XHTML) but now it's meant for annotations and mispellings, not for emphasis. So visually a `<u>` would have the same effect but it's probably not semantically correct.